packages/near-fast-auth-signer/src/api/index.ts

```diff
@@ -25,7 +25,7 @@ export const fetchAccountIds = async (publicKey: string): Promise<string[]> => {
     }
 
     try {
-      const res = await withTimeout(fetch(`${network.fastAuth.authHelperUrl}/publicKey/${publicKey}/accounts`), KIT_WALLET_WAIT_DURATION);
+      const res = await fetch(`${network.fastAuth.authHelperUrl}/publicKey/${publicKey}/accounts`);
```

Comment:
would like to know why withTimeout is removed on this line. Is it more preferred to wait longer until we get a response back from kit wallet?

Reply:
I think I forgot to rollback this code after my debugging. I will revert this changes. Good catch! :)